ices/86642.rs

```diff
@@ -0,0 +1,27 @@
+#![feature(impl_trait_in_bindings)]
```

Comment:
It seems like that this is likely a duplicate of the mentioned issue but let's see it with glacier.